### ISSUE TITLE
`$HOME/.ssh/authorized_keys` is centrally managed

### ISSUE THREAD

**Issue by ickc** [simonsobs-uk/data-centre-issue-tracker#5]
Currently, `$HOME/.ssh/authorized_keys` will be overwritten periodically by central management system. But this is counter-intuitive from the expectation of the [Filesystem Hierarchy Standard](https://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.html). A better approach would be putting it under `/etc` (Host-specific system configuration).

For example, use `AuthorizedKeysFile` in `/etc/ssh/sshd_config`:

```sh
# /etc/ssh/sshd_config
AuthorizedKeysFile /etc/ssh/%u/authorized_keys .ssh/authorized_keys
```

and put the centrally managed file in `/etc/ssh/$USER/authorized_keys` instead.

**Comment by rwf14f** [comment]
I agree, it would be better to change the configuration to centrally manage the authorized_keys file in /etc/ssh/.
You would still not be allowed to change it though. We would change the configuration to only use the file in /etc/ssh, it would replace ~/.ssh/authorized_keys, not amend it.
